Sources/Commands/SwiftPackageTool.swift

```diff
@@ -40,6 +61,8 @@ public enum PackageMode: Argument, Equatable, CustomStringConvertible {
             self = .fetch
         case "generate-xcodeproj":
             self = .generateXcodeproj
+        case "get-package-path":
+            self = .getPackagePath
```

Comment:
Its only needed for the functional tests for now and wasn't sure about the design so kept it undocumented. By directly you mean assume where the checkout is going to be located? That works for me